crmsh/corosync.py

```diff
@@ -167,56 +184,101 @@ def __init__(self, ip, port=5403, algo="ffsplit", tie_breaker="lowest",
         self.askpass = False
         self.cmds = cmds
         self.mode = mode
+        self.qnetd_ip = None
```

Comment:
What is the difference between `qnetd_addr` and `qnetd_ip`. We need to document what all of these variables are, some of them are not obvious enough to let them undocumented.

In fact, as far as I see, `self.qnetd_ip` is used only in 1 method. Maybe we could use it as a local variable. Os am I missing something?

Reply:
Thanks
I changed `self.qnetd_ip` as local variable